.travis.yml

```diff
@@ -188,6 +188,14 @@ matrix:
           stage: Cron tests
           env: SETUP_METHOD='apt'
 
+        # Regularly make sure that astropy can be used
```

Comment:
Comment incomplete - "in an Application bundle"?

Reply:
Oops thanks :)